extensions/wikia/AdEngine/js/provider/gpt/googleTag.js

```diff
@@ -152,6 +152,10 @@ define('ext.wikia.adEngine.provider.gpt.googleTag', [
 		return slot;
 	}
 
+	function refreshSlot(slot) {
+		window.googletag.pubads().refresh([slot]);
```

Comment:
Think of creating a follow-up to check the data we're afraid will stop working without using here AdEngine's `push` or maybe add it to AC of already existing follow-up ticket.